source/docs/application-notes/manual-pid-tuning.rst

```diff
@@ -191,9 +222,22 @@ The steps:
  6. Increase kS until the arm starts moving, then back off to just before that movement.
  7. Increase kP until you notice significant overshoot.
  8. Increase kD until the overshoot stops happening.
- 9. Repeat steps 7 and 8 until increasing kD results in more oscillation, or until the system oscillates on its way to the setpoint.
+ 9. Repeat steps 7 and 8 until increasing kD results in more oscillation, or until the system oscillates on its way to the setpoint. If oscillation on the way to setpoint is seen, decrease kD until it stops. If overshoot in general is happening and kD is already at max, reduce kP until it stops.
  10. Verify gains work for other setpoints as well. Tune kP/kD as appropriate for most general cases.
 
+.. dropdown:: "Why" for each step
+
+   1. We start with PID gains at 0 to isolate as many of the forces in play as possible, and iteratively get closer to the "ideal" gains.
+   2. The kG gain is meant to counteract the force of gravity, however the force of friction is also at play in an arm. The lowest possible kG that prevents the system from moving finds the lower bound of the gravity and friction component.
+   3. The highest possible kG that prevents the system from moving finds the upper bound of the gravity and friction component.
+   4. Setting kG to the middle point of the lower and upper bound is a good approximation for the true effect of gravity minus the force of friction.
```

Comment:
4. Setting kG to the middle point of the lower and upper bound is a good approximation for the true effect of gravity minus the force of friction.

to

4. Setting kG to the middle point of the lower and upper bounds is a good approximation for the true effect of gravity minus the force of friction.